x-pack/plugins/infra/public/alerting/metric_threshold/components/custom_equation/metric_row_controls.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiFlexItem, EuiButtonIcon } from '@elastic/eui';
+import { DELETE_LABEL } from '../../i18n_strings';
+
+interface MetricRowControlProps {
+  onDelete: () => void;
+  disableDelete: boolean;
+}
+
+export const MetricRowControls: React.FC<MetricRowControlProps> = ({ onDelete, disableDelete }) => {
```

Comment:
nit: Since we're not using `children`, I guess we could remove the usage of React.FC and just go for a functional component. WDYT?

```ts
export const MetricRowControls = ({ onDelete, disableDelete }: MetricRowControlProps) => {
```